package.json

```diff
@@ -65,7 +65,7 @@
     "terser-webpack-plugin": "^1.2.1",
     "uglify-template-string-loader": "^1.1.0",
     "webpack": "^4.28.4",
-    "webpack-babel-multi-target-plugin": "2.1.0-alpha.1",
+    "webpack-babel-multi-target-plugin": "^2.1.0-alpha.5",
```

Comment:
I would recommend pinning to the exact version in case of alpha releases, as in theory the API can be changed significantly before beta (although it's probably not the case). Will update the PR accordingly.

Reply:
True. Even `lit-element` is messing around with their API in rc releases.